__tests__/utils/fetchOriginalUrl.test.ts

```diff
@@ -0,0 +1,39 @@
+import fetchMock from 'jest-fetch-mock';
+import fetchOriginalUrl from '../../src/utils/fetchOriginalUrl';
+import { act } from 'react-dom/test-utils';
+
+fetchMock.enableMocks();
+
+describe('fetchOriginalUrl', () => {
+    const shortUrlCode = '442d39ac';
+    const originalUrl = 'https://github.com/Real-Dev-Squad/tiny-site-frontend/pull/40';
+
+    it('fetches and displays the original URL if the response is successful', async () => {
+        const responseData = { url: { OriginalUrl: originalUrl } };
+
+        fetchMock.mockResponse(JSON.stringify(responseData), { status: 200 });
```

Comment:
the `fetchMock` returns a string or null but this is a JSON

Reply:
https://github.com/Real-Dev-Squad/tiny-site-frontend/pull/43

Reply:
No, the fetchMock returns a stringified JSON. We need to parse it to get the JSON object.